### PR TITLE
fix(docs): update installation documentation links

### PR DESCRIPTION
**Pull request by jniles**
This commit improves the installation script by installing the latest
LTS of NodeJS.  It also adds relevant links and makes the README link to
the docs.bhi.ma website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ issues are a decent place to start.
 
 Installation
 -------------------
-See the [installation guide](./docs/en/for-developers/installing-bhima.md).
+See the [installation guide](https://docs.bhi.ma/en/for-developers/installing-bhima.html).
```
Comment (mbayopanda) [inline README.md:46]: We hope the domain will never change 😄 and if it is the case, we will change this line again...

File: docs/en/for-developers/installing-bhima.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ This guide will get you up and running with bhima locally. Please note that bhim
 Before you begin the installation process, please make sure you have all the bhima dependencies installed locally. We only test on Linux, so your best bet is to use a Linux flavor you are familiar with. Please make sure you have recent version of:
 
 1. [MySQL](http://dev.mysql.com/downloads/) \(5.6 or newer\)
-2. [Redis](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/redis.io)
+2. [Redis](https://github.com/IMA-WorldHealth/bhima/blob/master/docs/redis.io)
```
Comment (mbayopanda) [inline docs/en/for-developers/installing-bhima.md]: the `Redis` link must point to https://redis.io

File: docs/fr/for-developers/installing-bhima.md [hunk 3]
```diff
@@ -9,11 +9,11 @@ Ce guide vous permettra de vous familiariser avec bhima localement. Veuillez not
 Avant de commencer le processus d'installation, assurez-vous que toutes les dépendances bhima sont installées localement. Nous ne testons que sous Linux. Il est donc préférable d’utiliser une version de Linux que vous connaissez bien. Assurez-vous d'avoir la version récente de:
 
 1. [MySQL](http://dev.mysql.com/downloads/) \(5.6 ou plus récent \)
-2. [Redis](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/redis.io)
+2. [Redis](https://github.com/IMA-WorldHealth/bhima/blob/master/docs/redis.io)
```
Comment (mbayopanda) [inline docs/fr/for-developers/installing-bhima.md]:
```
[Redis](https://redis.io)
```

**Review by mbayopanda — COMMENTED** [review]
This PR LGTM, there are just links to redis web site which must updated

**Comment by jniles** [comment]
@mbayopanda, fixed!

**Pull request merged**